measures/ResStockArguments/resources/constants.rb

```diff
@@ -100,6 +101,7 @@ def self.Auto
 
   def self.arguments_to_register
     # list of resstock arguments; reported as build_existing_model.<argument_name>, ...
-    return []
+    return ['hvac_control_heating_season_period',
+            'hvac_control_cooling_season_period']
```

Comment:
Instead of registering available season periods (e.g., Oct 25 - Oct 21), register the unavailable period (e.g., Oct 22 - Oct 24). And then for available all year, register "Never". Call registered arguments something like `xxx.hvac_control_heating_unavailable_period`.